zuul/layout.yaml

```diff
@@ -71,6 +71,24 @@ pipelines:
     failure:
       gerrit: {}
 
+  - name: experimental-asr-prll
+    description: >
+      Patchsets will enter this pipeline when a cisco-exp-asr-prll comment is
+      left on them. This pipeline is for running experimental ASR1K jobs in parallel
+      for devs to test their changes to job code without impacting the ordinary
+      running of the CI.
+    success-message: Build succeeded (experimental-asr-prll pipeline).
+    failure-message: Build failed (experimental-asr-prll pipeline).
+    manager: IndependentPipelineManager
+    trigger:
+      gerrit:
+        - event: comment-added
+          comment: (?i)^(Patch Set [0-9]+:)?( [\w\\+-]*)*(\n\n)?\s*cisco-exp-asr-prll
+    success:
+      gerrit: {}
+    failure:
+      gerrit: {}
```

Comment:
There shouldn't be a need to add an additional queue for this new job, the normal experimental pipeline should be fine, the new "Cisco-CI-Experimental-Regex:" concept has merged, so adding: 

`Cisco-CI-Experimental-Regex: .*-asr1k-prll.*`

in your commit message to gerrit, will limit the number of jobs that get run for cisco-experimental

Reply:
done